README.md

```diff
@@ -286,6 +286,12 @@ create_table :widgets do |t|
 end
 ```
 
+### `#add_reference(table, ref_name, **extra_args)`
+Adds a new reference column. Nandi will validate that the foreign key flag is not set to true; use `add_foreign_key` and `validate_foreign_key` instead! Nandi will also set the `index: false` flag, as this is unsafe. Add an index in a separate migration.
```

Comment:
when you say "this is unsafe", does this refer to when index is true? the sentence here sounds a little misleading to me.

Reply:
`index` defaults to `true`, so it's unsafe by default.

Reply:
Oh, I see what you're getting at ...